pages/common/psgrep.md

```diff
@@ -0,0 +1,15 @@
+# psgrep
```

Comment:
Did you mean `pgrep`? `psgrep` isn't a thing on my Linux laptop here, but that could just be my distribution.

Reply:
`pgrep` is a separate command from `psgrep`.  To quote the manpage: 

```
psgrep  is  a small Bash shell script that searches the process list (as obtained by ps(1) ) using the awesome utility grep(1) for its power.  It is more
       useful than pgrep(1) because not only can it search the process list and return a PID, it can give more useful information such as  its  UID,  GID,  mem-
       ory/CPU usage, niceness, and anything else supported by ps.
```

Reply:
I'm running MacOS High Sierra.  The manpage doesn't specify that this is for OSX but, your comment and @agnivade makes me question.